packages/rpc-graphql/src/loaders/account.ts

```diff
@@ -24,7 +24,7 @@ function normalizeArgs({
 /* Load an account from the RPC, transform it, then return it */
 async function loadAccount(rpc: Rpc, { address, ...config }: ReturnType<typeof normalizeArgs>) {
     const account = await rpc
-        .getAccountInfo(address, config as Parameters<SolanaRpcMethods['getAccountInfo']>[1])
+        .getAccountInfo(address, config as Parameters<GetAccountInfoApi['getAccountInfo']>[1])
```

Comment:
This still typechecks if you delete this. Maybe you should pare these down and only leave the problem ones.